lib/moonshot/stack.rb

```diff
@@ -176,14 +176,35 @@ def get_stack(name)
       raise "Could not describe stack: #{name}"
     end
 
+    def upload_template_to_s3
+      raise 'The S3 bucket to store the template in is not configured.' unless @config.template_s3_bucket # rubocop:disable LineLength
```

Comment:
Please reformat this to a multi-line conditional, rather than disabling rubocop.